### DISCUSSION TITLE
How to access postgres from host

### DISCUSSION THREAD

**Discussion by Itisfilipe** [silviogutierrez/reactivated#298] [Q&A] [answered]
How can I access Postgres dev database with something like postico? It might be related to nNxOs but I just can't figure it out 🤔

**Comment by silviogutierrez** [comment, marked as answer]
So it uses a socket file to avoid binding to ports. To see the right socket file, just run `export` inside the activated shell and you should see PGHOST.

You can then connect to that. It should not need any authentication. The database name is `database` and you may need to specify port `1` if required.

Example:

```
[nix-shell:~/Sites/silviogutierrez/silviogutierrez.com/main]$ export | grep PGHOST
declare -x PGHOST="/tmp/reactivated/ff657f1d3a00ad59e7833de1de612408"
```

And using TablePlus:

<img width="553" alt="image" src="https://github.com/silviogutierrez/reactivated/assets/92824/b5ffc727-85c0-42c4-98a9-057a0a3d4640">
